server/src/resolvers/employee.resolver.ts

```diff
@@ -71,7 +71,7 @@ class EmployeeResolver {
       { key },
       { relations: ["employees"] }
     );
-
+    console.log("req.session", req.session);
```

Comment:
Lets remove logging from the code please

Reply:
I realize this wasn't you here but if you can please remove that would be great.